build.gradle

```diff
@@ -117,6 +119,9 @@ subprojects {
             compileTestJava {
                 options.encoding = 'UTF-8'
                 options.compilerArgs << '-Xlint:unchecked' << '-Xlint:deprecation'
+
+                sourceCompatibility = 11
```

Comment:
Would it make sense to do something like this?
```
ext.javaTestTargetVersion = JavaVersion.VERSION_11
...
sourceCompatibility = javaTestTargetVersion
sourceCompatibility = javaTestTargetVersion
```
or
```
sourceCompatibility = JavaVersion.VERSION_11
sourceCompatibility = JavaVersion.VERSION_11
```

Reply:
I'm not sure one is better than the other, but consistency would be good. I've updated it to align to your suggestion since that was already in use in some places.